README.md

```diff
@@ -12,6 +12,14 @@ Please put all issues regarding IPFS _design_ in the
 [ipfs repo issues](https://github.com/ipfs/ipfs/issues).
 Please put all issues regarding go IPFS _implementation_ in [this repo](https://github.com/ipfs/go-ipfs/issues).
 
+## Security Issues
```

Comment:
I think that should just be an issue, it doesn't look like more than a TODO. Agreed. Will PR it's removal now.